Sources/Layouts/TextViewLayout.swift

```diff
@@ -130,6 +130,10 @@ open class TextViewLayout<TextView: UITextView>: BaseLayout<TextView>, Configura
     // MARK: - overriden methods
 
     open override func configure(view textView: TextView) {
+        /// `isSelectable` is default to false, but it allows overriding in `configure` block
+        /// By enabling it, it may have side effects
```

Comment:
I only tested it on iOS 10/11 and I haven't fully tested on iOS 8/9. Let me remove this.